cmd/spire-server/cli/run/run.go

```diff
@@ -109,6 +105,14 @@ type caSubjectConfig struct {
 	UnusedKeys   []string `hcl:",unusedKeys"`
 }
 
+type bundleEndpointConfig struct {
+	Enabled    bool                      `hcl:"enabled"`
```

Comment:
How would you feel about not having this? In my opinion, it should be implied by the `bundle_endpoint` section existing.

Reply:
Sounds good to me!